Package.swift

```diff
@@ -17,18 +17,20 @@ import PackageDescription
 
 let package = Package(
     name: "async-http-client",
+    platforms: [.iOS(.v12), .tvOS(.v12)],
```

Comment:
why do we need this? older platforms should be covered by `#if canImport(Network)` and `if #available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)`

Reply:
There is no valid code path for iOS 11.0 or less. If we don't have this it'll try to use a non TS path on iOS 11 which isn't valid.

Reply:
However, it's no more broken than it is today, and adding these constraints is a semver major.

Reply:
removed